ROX-32265: Update docs for base image detection (#106054)
openshift/openshift-docs: modules/view-base-imageinfo-portal.adoc

// Module included in the following assemblies:
//
// * operating/examine-images-for-vulnerabilities.adoc
:_mod-docs-content-type: PROCEDURE
[id="view-base-imageinfo-portal_{context}"]
= View base layer information for an image in the {product-title-short} portal

[role="_abstract"]

> Base images are trusted images that DevOps users can include when building applications. {product-title-short} provides information about base images, such as the base image name, digest, and age. You can use the {product-title-short} to view this information.

Reviewer: ```suggestion
{product-title-short} can provide information about the base images identified in your applications, such as the base image name, digest, and age. You can use the {product-title-short} to view this information.
```

Also, should we mention this only shows if a base image was detected?

// Module included in the following assemblies:
//
// * operating/examine-images-for-vulnerabilities.adoc
:_mod-docs-content-type: PROCEDURE
[id="view-base-imageinfo-portal_{context}"]
= Viewing base image information for an image in the {product-title-short} portal

[role="_abstract"]

{product-title-short} can provide information about the base images identified in your application, such as the base image name, digest, and age. You can use the {product-title-short} portal to view this information.

.Procedure

. Select *Vulnerability Management* -> *Results*.
. Click *<number> Images*. 
. Optional: To filter the display to only show base images, take the following actions:
.. In the filter bar, select *Image component* and *Layer type*.
.. In the filter bar, click *Filter by Layer type* and click *Base image*. 
. Click the name of the image that you want to examine. 
. Click the arrow to expand *Base image assessment*. The image name, digest, and image age are shown. 
+
[NOTE]
====

> The information displayed in this section shows only that a base image was detected. To view vulnerabilities by application layer, you need to use the filter bar to filter CVEs by layer type. See "View vulnerabilities in a base image layer".

Reviewer: ```suggestion
The information displayed in this section is presented only when a base image was detected. To view vulnerabilities by application layer, you need to use the filter bar to filter CVEs by layer type. See "View vulnerabilities in a base image layer". If the system did not detect any base image, all layers are considered "application" layers.
```
